aio/content/examples/animations/src/app/hero-list-page.component.ts

```diff
@@ -52,13 +54,11 @@ export class HeroListPageComponent implements OnInit {
   @HostBinding('@pageAnimations')
   public animatePage = true;
 
-  _heroes = [];
 // #docregion filter-animations
   heroTotal = -1;
 // #enddocregion filter-animations
-  get heroes() {
-    return this._heroes;
-  }
+  get heroes() { return this._heroes; }
+  private _heroes = [];
```

Comment:
Instead of adding tslint:disable, this looks like it could be fixed by just doing
```suggestion
  readonly heroes = [];
```

Reply:
The problem with that is that we would then have to make changes to the rest of the component to be able to update the `heroes` property from inside the class (i.e. [here][1] and [there][2]), which felt outside the scope of this PR.

While it is possible to refactor the component to avoid these problems, it is probably outside the scope of this PR.

[1]: https://github.com/angular/angular/blob/9308804ae024529fddbe5c9bfc1883ef4dd4af93/aio/content/examples/animations/src/app/hero-list-page.component.ts#L64
[2]: https://github.com/angular/angular/blob/9308804ae024529fddbe5c9bfc1883ef4dd4af93/aio/content/examples/animations/src/app/hero-list-page.component.ts#L70